temperature/router.py

```diff
@@ -0,0 +1,48 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.future import select
+from datetime import datetime
+
+from . import models, schemas, crud
```

Comment:
Ensure that the `models` module contains the `City` and `Temperature` models, which are used for database operations related to temperature records.